sdk/storage/CMakeLists.txt

```diff
@@ -9,19 +9,37 @@ set(CMAKE_CXX_STANDARD_REQUIRED True)
 
 option(BUILD_STORAGE_SAMPLES "Build sample codes" ON)
 
-set(AZURE_STORAGE_HEADER
```

Comment:
I don't think it's necessary at all to define so many variables like `AZURE_STORAGE_BLOB_HEADER`, `AZURE_STORAGE_ADLS_HEADER`, `AZURE_STORAGE_FILE_HEADER` and `AZURE_STORAGE_ADLS_SOURCE` etc.

Why do those common files go with `AZURE_STORAGE_BLOB_HEADER` rather than `AZURE_STORAGE_ADLS_HEADER` or `AZURE_STORAGE_FILE_HEADER`

Reply:
on a second thought, this should be fine.

Reply:
1. It is for future support of building/installing only Blob/ADLS, etc.
2. Common files should go with all individual services. Not only going to blob.